### PR TITLE
fix: project orm tests

### PR DESCRIPTION
**Pull request by nbeteram**
The orm project tests are done.

### CODE REVIEW

File: backend/orm_functions/project.ts [hunk 1]
```diff
@@ -357,6 +357,8 @@ export async function filterProjects(
             );
 
             for (const c of project_found[0].project_role) {
+                console.log(c._count.contract);
+                console.log(c.positions);
```
Comment (BramDevlaminck) [inline backend/orm_functions/project.ts:361]: we should remove these log statements

**Review by BramDevlaminck — CHANGES_REQUESTED** [review]
🚨 looks good except the console.log police has caught you red-handed ;) 🚨

File: backend/orm_functions/project.ts (hunk 1 above)
Comment (nbeteram) [reply backend/orm_functions/project.ts]: This has been resolved

**Review by BramDevlaminck — APPROVED** [review]
great

**Review by jay-tux — APPROVED** [review]
🪨

**Pull request merged**